style.css

```diff
@@ -0,0 +1,138 @@
+body {
+  margin: 0;
+  padding: 0;
```

Comment:
put this margins and padding in the universal selector like
* {
margin : 0;
padding : 0;
box-sizing: border-box;
}

by putting this code you can remove all margin 0 and padding 0 from rest of the code...

the code shown above is rule for reset the marging and padding property in css 
see some documentation about "CSS reset rules"...

Reply:
yeah sure, I will see some documentation 
I made these changes